package/CHANGELOG

```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/??  richardjgowers, IAlibay, tylerjereddy, xiki-tempula,
+??/??/??  richardjgowers, IAlibay, tylerjereddy, xiki-tempula, lilyminium
 
   * 1.0.2
```

Comment:
To avoid confusion can you change this one to 1.1.0 please?
```suggestion
  * 1.1.0
```

We'll change the version tags on release, but this will just confuse folks doing any PRs.

Reply:
Ok, done :)